src/stories/decorators/FillHeight.tsx

```diff
@@ -1,11 +1,5 @@
-import type { Story } from '@storybook/react';
-
-function FillHeight(MyStory: Story) {
-  return (
-    <div style={{ display: 'grid', height: '100vh' }}>
-      <MyStory />
-    </div>
-  );
+function FillHeight<T>(story: () => T) {
+  return <div style={{ display: 'grid', height: '100vh' }}>{story()}</div>;
```

Comment:
This decorator and a couple of others were causing the source blocks in the stories' documentation to show `<No Display Name />` instead of the story source code.